src/app/app.component.ts

```diff
@@ -1,18 +1,18 @@
-import { Component } from '@angular/core';
+import { Component } from "@angular/core";
```

Comment:
Dont we want these as singular? I dont care tho, as long as they wont cause merge issues for others